docs/source/install.rst

```diff
@@ -172,7 +175,7 @@ Test Dask with ``py.test``::
     cd dask
     py.test dask
 
-Installing Dask naively may not install all requirements by default (see the `Pip`_ section above).
+Installing Dask naively may not install all requirements by default (see the `Pip` section above).
```

Comment:
Removing the `_` here also removes this as a link to the pip installation section above (currently rendered as "Pip" in italics)

Reply:
Yeah, we no longer have the section above.   I'll remove the accidental italics.